From db68059fbbb7883b3c1a4f645e0b555e79833bcf Mon Sep 17 00:00:00 2001
From: H1mJT <himjyoti22@gmail.com>
Date: Tue, 7 Mar 2023 19:32:35 +0530
Subject: [PATCH] Fix OLDPWD not set

---
 setup/install_android_sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup/install_android_sdk.sh b/setup/install_android_sdk.sh
index 8a85cdb..a21ea6a 100755
--- a/setup/install_android_sdk.sh
+++ b/setup/install_android_sdk.sh
@@ -21,7 +21,7 @@ function setup_android_sdk() {
         yes | "${SDK_DIR}"/cmdline-tools/bin/sdkmanager --sdk_root="${SDK_DIR}" "${package:?}"
     done < "${CUR_DIR}"/setup/android-sdk-minimal.txt
     rm /tmp/tools.zip
-    cd - || exit
+    cd .. || exit
 }
 
 setup_android_sdk
-- 
2.37.1.windows.1

